pxtblocks/fields/field_imagedropdown.ts

```diff
@@ -93,6 +93,9 @@ export class FieldImageDropdown extends FieldDropdown implements FieldCustom {
                 case "Enter":
                 case "Space": {
                     this.buttonClick_(this.buttons[this.activeDescendantIndex].getAttribute('data-value'));
+                    e.preventDefault();
+                    e.stopPropagation();
+                    return;
```

Comment:
It would have thrown an error if there a was a break statement here..., but this is now correct.